feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/ui/CommonClaims.kt

```diff
@@ -51,77 +51,26 @@ internal fun CommonClaims(
             Row(
               modifier = Modifier
                 .padding(22.dp)
+                .align(Alignment.Start)
                 .fillMaxWidth(),
+              horizontalArrangement = Arrangement.SpaceBetween,
             ) {
-              Box(
-                modifier = Modifier
-                  .size(16.dp)
-                  .clip(CircleShape)
-                  .background(Color(claim.color()))
-                  .align(Alignment.CenterVertically),
-              )
-
-              Spacer(modifier = Modifier.padding(12.dp))
-
               Text(claim.displayName, Modifier.align(Alignment.CenterVertically))
+              Icon(
+                painter = painterResource(id = com.hedvig.android.odyssey.R.drawable.ic_list_item_indicator),
+                contentDescription = "Arrow",
+              )
+            }
+            if (index < commonClaims.size - 1) {
```

Comment:
A nice convenience Kotlin offers is being able to change this with `lastIndex`. Exactly like it's used here https://github.com/HedvigInsurance/android/blob/fbe54efb7b913ae465b3edda3b8f4cd8ab23a67d/feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/step/location/LocationDestination.kt#L127-L129
```suggestion
            if (index != commonClaims.lastIndex) {
```

Reply:
NIce :D